apps/modernization-ui/src/styles/_tooltip.scss

```diff
@@ -5,7 +5,6 @@ $bubbleWidth: calc($arrowWidth + 100%);
 
 [data-tooltip-position][aria-label] {
     position: relative;
-    z-index: 1000;
```

Comment:
Issue: this is needed for icons that appear beneath the `DataTable`

Reply:
It was an issue earlier, but just reverted it and it seems to have fixed the bug so will make that change, thank you